docker/local/README.md

```diff
@@ -0,0 +1,39 @@
+## Turbinia local stack using Docker
+Turbinia can be run locally without any Cloud components using Docker. It will use Redis, Celery and local disk to store data and perform message broker functionality.
+
+### Caveats
+rawdisk: As Turbinia uses the loop device to mount different types of evidence (eg raw disks) the host operating system should support the loop device. Linux is currently the only OS that supports the processing of raw disks. It is possible to process different evidence types, eg compresseddirectory, using the local stack on OSX or Windows.
+
+googleclouddisk: Turbinia as local stack can currently not process Google Cloud disks.
+
+### Running
+
+#### Step 1
+Checkout the Turbinia source code. If you intend to start developing please fork the repository on github first and check out your own forked instance.
+```
+$ git clone https://github.com/google/turbinia.git
+$ cd turbinia
+```
+#### Step 2
+Copy the configuration template for the local stack to the ```./conf`` folder (create the folder if it does not exist).
+```
+$ cp docker/local/turbinia-local-template.conf ./conf/turbinia.conf
+```
+#### Step 3
+Let's bring up the local Turbinia stack
+```
+$ docker-compose -f ./docker/local/docker-compose.yml up
+```
+Redis, a Turbinia server and worker should now be running on your local system and a local persistent 'evidence' folder will have been created containing the Turbinia log file and processing output.
```

Comment:
Very cool.

Reply:
Can we mention somewhere what the status is if someone wants to run more than one worker on a separate machine?  Even if it's not supported yet, it would be good to mention it.  If it is supported, it might be good to add a comment about what  assumptions about the network topology are being made.

Reply:
It's supported (afaik....) and super duper easy :). Just duplicate the turbinia-worker block in the docker-compose file and you have a second worker. Let me add one and comment it out. 
docker compose will take care of the network, there is nothing to setup for the user, it's transparent.

Reply:
mmh, if you mean running another worker on *another* machine that's possible but would need 2 things:
* change 'expose' to 'ports' in docker-compose redis configuration so it's exposed on the host and other hosts can connect
* enable shared storage (through NFS) instead of the currently used shared volumes that are only accesible from within the docker-compose network

Not hard to setup, probably takes 30 minutes.

Reply:
Yeah, shared storage is a prerequisite for any local/hybrid setup.  Maybe we should take a second pass later in a separate PR to add instructions for workers on separate machines?  Then we can also address the networking/security related bits too?

Reply:
Agree, opening issue for that.